SmartDeviceLink/SDLStreamingVideoLifecycleManager.m

```diff
@@ -774,6 +781,10 @@ - (BOOL)isHmiStateVideoStreamCapable {
     return @[h264raw, h264rtp];
 }
 
+- (NSString *)videoStreamBackgroundString {
```

Comment:
This should probably be a getter only property.

Reply:
Fixed